### ISSUE TITLE
Small issue: Hard to see black words in dark mode

### ISSUE THREAD

**Issue by jennicao** [dilevin/computer-graphics-raster-images#70]
I missed this at first, but I couldn't see the grayscale formula in the README in dark mode. If it isn't too much trouble, I was hoping all assignments could be checked for dark mode compatibility? It's fine if that's not possible too though!

<img width="970" alt="image" src="https://github.com/user-attachments/assets/c1a71baa-f94e-4201-afbb-da50c2e1482b">

**Comment by Zhecheng-Wang** [comment]
I agree, but AFAIK the equations are embedded as images, so can't really change the color.
